backend/schema.js

```diff
@@ -14,71 +15,151 @@ const typeDefs = gql`
+    updatedByUserId: ID
+    members: [Member]
+    jobsAppliedTo: [Job]
+    tags: [ScoredTag]
   }
 
   type Member {
-    User: User!
-    Role: RoleType!
-    NoticePeriod: Int
-    AvailableAt: Date
+    user: User!
+    role: RoleType!
+    noticePeriod: Int
+    availableAt: Date
   }
 
   type User {
-    Id: ID!
-    Available: Boolean!
-    FirstName: String! 
-    LastName: String! 
-    Description: String! 
-    SocialAccounts: [SocialAccount]
-    CreatedAt: Date!
-    CreatedByUserId: ID!
-    UpdatedAt: Date
-    UpdatedByUserId: ID
-    TeamMemberOf: [Team]
-    JobsCreated: [Job]
-    Tags: [ScoredTag]
+    id: ID!
+    available: Boolean!
+    firstName: String! 
+    lastName: String! 
+    description: String! 
+    socialAccounts: [SocialAccount]
+    createdAt: Date!
+    createdByUserId: ID!
+    updatedAt: Date
+    updatedByUserId: ID
+    teamMemberOf: [Team]
+    jobsCreated: [Job]
+    tags: [ScoredTag]
   }
 
   type Job {
-    Id: ID!
-    Name: String!
-    Header: String
-    Description: String
-    Applications: [Team]
-    CreatedAt: Date!
-    CreatedByUserId: ID!
-    UpdatedAt: Date
-    UpdatedByUserId: ID
-    Tags: [Tag]
+    id: ID!
+    name: String!
+    header: String
+    description: String
+    applications: [Team]
+    createdAt: Date!
+    createdByUserId: ID!
+    updatedAt: Date
+    updatedByUserId: ID
+    tags: [Tag]
   }
 
   type SocialAccount {
-    ExternalId: String!
-    Type: SocialAccountType!
+    externalId: String!
+    type: SocialAccountType!
   }
  
   type ScoredTag {
-    Name: String!
-    Score: Int
+    name: String!
+    score: Int
   }
 
   type Tag {
-    Name: String!
+    name: String!
   }
 
-  type Query {
-    allUsers: [User]
+  #QUERIES#####################################
+
+  #based on token
+  query {
+    me: User
+  }
+
+  query Team($id: Int) {
+    team(id: $id): Team
+  }
+
+  query Job($id: Int) {
+    job(id: $id): Job
+  }
+
+  query MatchingTags ($search: String, $first: Int, $cursor: String) {
+    matches(search: $search, first: $first, cursor: $cursor): {
+      totalCount
+      edges {
+        node: Tag
+        cursor
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+
+  query MatchingJobs ($tags: Tag[], $first: Int, $cursor: String) {
+    matches(tags: $tags, first: $first, cursor: $cursor): {
+      totalCount
+      edges {
+        node: Job
+        cursor
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+
+  query MatchingTeamSummaries ($tags: Tag[], $first: Int, $cursor: String) {
+    matches(tags: $tags, first: $first, cursor: $cursor): {
+      totalCount
+      edges {
+        node: TeamSummary
+        cursor
+      }
+      pageInfo {
+        endCursor
+        hasNextPage
+      }
+    }
+  }
+
+  fragment TeamSummary on Team {
+    id
+    name  
+    header
+    description
+    tags
+  }
+
+  #MUTATIONS###################################
```

Comment:
Naming wise I tend to follow [this convention](https://blog.apollographql.com/designing-graphql-mutations-e09de826ed97), although its not needed at all for this project, but so you know :) In the end everything looks like:
```
type RootMutation {
  createTodo(input: CreateTodoInput!): CreateTodoPayload
  toggleTodoCompleted(input: ToggleTodoCompletedInput!): ToggleTodoCompletedPayload
  updateTodoText(input: UpdateTodoTextInput!): UpdateTodoTextPayload
  completeAllTodos(input: CompleteAllTodosInput!): CompleteAllTodosPayload
}
```